R/config.R

```diff
@@ -188,7 +188,7 @@ config_new <- function(path_archive, use_file_store, require_complete_tree) {
 
 
 config_serialise <- function(config, path) {
-  config$schemaVersion <- scalar(config$schemaVersion) # nolint
+  config$schema_version <- scalar(config$schema_version) # nolint
```

Comment:
I think this `# nolint` can go now, it was about the non-snake-case name

```suggestion
  config$schema_version <- scalar(config$schema_version)
```